IdentityCore/src/webview/systemWebview/session/MSIDSystemWebviewController.m

```diff
@@ -237,10 +243,11 @@ - (void)dismiss
         return safariController;
     }
 #else
-    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Couldn't create session on macOS. Safari allowed flag %d", safariAllowed);
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, nil, @"Couldn't create session on macOS or visionOS. Safari allowed flag %d", safariAllowed);
```

Comment:
if target OS is Vision, we are returning nil at line 230, so is this log change correct ?

Reply:
That's true, I added the visionOS logic above later. I'll add a log line up there too